### DISCUSSION TITLE
Jumping while typing

### DISCUSSION THREAD

**Discussion by o777l** [winstxnhdw/lc-hax#433] [General]
After using the `/jump` command I can't type in the in game chat without jumping up and down everytime pressing space bar. It looks really stupid 😂

**Comment by winstxnhdw** [comment]
@xAstroBoy

**Reply by xAstroBoy** [reply]
i fixed it in the new camera PR, is a minor fix really.

**Reply by winstxnhdw** [reply]
I don't see the file change.

**Reply by xAstroBoy** [reply]
https://github.com/xAstroBoy/lc-hax/blob/New-Camera/lc-hax/Scripts/Patches/UnlimitedJumpPatch.cs

**Comment by o777l** [comment]
> https://github.com/xAstroBoy/lc-hax/blob/New-Camera/lc-hax/Scripts/Patches/UnlimitedJumpPatch.cs

I searched for some kind of file related to jump through windows 10 many times I did not find it. Why is search so bad on windows

Another thing, is it possible to increase the speed of it. As in, if you press space it will instantly perform double jump? I guess we  are getting into fly mods here, but I don't enjoy noclip so

**Reply by xAstroBoy** [reply]
Just keep pressing the jump button…

**Reply by o777l** [reply]
Well dontcha know there was a delay and I fell in between
